### DISCUSSION TITLE
How to enter INI ...+2W6

### DISCUSSION THREAD

**Discussion by linkrope** [Lector/DSA5maptool#18] [Q&A] [answered]
"Höhlenspinne" has an INI of 13+2W6. With the Tokeneditor, I can enter the 13, but how do I change +1W6 into +2W6?

**Comment by Lector** [comment, marked as answer]
It is currently not possible to input multiple d6 in a token's initiative.

As a workaround, you can utilize the 'multi-roll initiative' button, which is the first button on the GM sheet. For example, you can input '2d6+13' in this field:
![image](https://github.com/Lector/DSA5maptool/assets/765587/60c4263d-8218-4a90-8ae8-719d9646d3d7)
![image](https://github.com/Lector/DSA5maptool/assets/765587/97ab2890-a46b-4595-9e71-456cf698692d)

I was not initially aware that there are creatures in the 5th edition with multiple dice for initiative. In the next version, I plan to expand the character sheets to accommodate this information on the tokens.

**Comment by Lector** [comment]
This feature is now available in v1.4.0
